components/block/element.js

```diff
@@ -53,28 +50,95 @@ export default class Block extends LitElement {
     return this.cached_parent;
   }
 
+  get parentName() {
+    if (!Boolean(this.parent)) return;
+
+    const parentId = this.model.attributes.parent_block_id;
+    const parentObject = this.layout.blocks.findWhere({
+      id: parentId,
+    });
+
+    return this.formatViewTypeName(
+      parentObject.attributes.definition_identifier
+    );
+  }
+
+  get viewTypeName() {
+    return this.formatViewTypeName(this.model.attributes.definition_identifier);
+  }
+
   get isInLinkedZone() {
     return this.model.zone().is_linked();
   }
 
+  formatViewTypeName(name) {
+    if (!Boolean(name)) return;
```

Comment:
You don't need Boolean here

Reply:
Done https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/41b03365f396fa39af17bf90f4911f75540e1d01